modules/ocf_desktop/manifests/firefox.pp

```diff
@@ -0,0 +1,19 @@
+class ocf_desktop::firefox {
+  package {
+    'firefox':
+      ensure => present;
+    'iceweasel':
+      ensure => absent;
+  }
+
+  file {
+    # disable caching, history, blacklisting, and set homepage
+    '/etc/firefox/profile/prefs.js':
+      content => template('ocf_desktop/firefox/prefs.js.erb'),
+      require => Package['firefox'];
+    # start maximized by default
+    '/etc/firefox/profile/localstore.rdf':
+      source  => 'puppet:///modules/ocf_desktop/firefox/localstore.rdf',
+      require => Package['firefox']
+  }
+}
```

Comment:
@chriskuehl I agree. Will move to `ocf::packages::firefox` for consistency with Chrome. However, ocf_tv and servers don't include mozilla.debian.net and google-chrome sources.list, so it won't work in extrapackages.